bin/main.rb

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env ruby
```

Comment:
Your main.rb file should puts "Hello world" ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from Pathwright)

Reply:
Oh, sori. we must have missed that. Thanks for pointing it out